Dockerfile.dist

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:wily
+
+ENV DNSMASQ_VERSION 2.75-1
+ENV GIT_VERSION 1:2.5.0-1
+
+RUN apt-get update && \
+    apt-get install -y \
+    dnsmasq=${DNSMASQ_VERSION} \
+    git=${GIT_VERSION} && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+RUN mkdir -p /etc/mayu /var/lib/mayu /usr/lib/mayu
+COPY mayu mayuctl /usr/bin/
+COPY tftproot /usr/lib/mayu/tftproot
+COPY images /usr/lib/mayu/images
```

Comment:
@hectorj2f the dockerfile moved into the bin-dist folder and we need to create the docker image from inside there. reason: you can fetch the released tarball and create your own docker image (with your Mayu config) from it.